controls/cis_al2023.yml

```diff
@@ -0,0 +1,2166 @@
+      - audit_rules_dac_modification_chmod
+      - audit_rules_dac_modification_chown
+      - audit_rules_dac_modification_fchmod
+      - audit_rules_dac_modification_fchmodat
+      - audit_rules_dac_modification_fchown
+      - audit_rules_dac_modification_fchownat
+      - audit_rules_dac_modification_fremovexattr
+      - audit_rules_dac_modification_fsetxattr
+      - audit_rules_dac_modification_lchown
+      - audit_rules_dac_modification_lremovexattr
+      - audit_rules_dac_modification_lsetxattr
+      - audit_rules_dac_modification_removexattr
+      - audit_rules_dac_modification_setxattr
+
+  - id: 5.2.3.10
+    title: Ensure successful file system mounts are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_media_export
+
+  - id: 5.2.3.11
+    title: Ensure session initiation information is collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_session_events
+
+  - id: 5.2.3.12
+    title: Ensure login and logout events are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_login_events_faillock
+      - audit_rules_login_events_lastlog
+      - var_accounts_passwords_pam_faillock_dir=run
+
+  - id: 5.2.3.13
+    title: Ensure file deletion events by users are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_file_deletion_events_rename
+      - audit_rules_file_deletion_events_renameat
+      - audit_rules_file_deletion_events_unlink
+      - audit_rules_file_deletion_events_unlinkat
+
+  - id: 5.2.3.14
+    title: Ensure events that modify the system's Mandatory Access Controls are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_mac_modification
+      - audit_rules_mac_modification_usr_share
+
+  - id: 5.2.3.15
+    title: Ensure successful and unsuccessful attempts to use the chcon command are recorded (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_execution_chcon
+
+  - id: 5.2.3.16
+    title: Ensure successful and unsuccessful attempts to use the setfacl command are recorded (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_execution_setfacl
+
+  - id: 5.2.3.17
+    title: Ensure successful and unsuccessful attempts to use the chacl command are recorded (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_execution_chacl
+
+  - id: 5.2.3.18
+    title: Ensure successful and unsuccessful attempts to use the usermod command are recorded (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_privileged_commands_usermod
+
+  - id: 5.2.3.19
+    title: Ensure kernel module loading, unloading and modification is collected (Automated)
```

Comment:
Replace with "title: Ensure kernel module loading unloading and modification is collected (Automated)". Missing comma.